phoenix5-spark3/pom.xml

```diff
@@ -183,6 +183,15 @@
             </ignoredDependencies>
           </configuration>
         </plugin>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-javadoc-plugin</artifactId>
+            <configuration>
+                <skip>${skip.spark.javadoc}</skip>
+            </configuration>
+        </plugin>
     </plugins>
   </build>
+    <profiles>
+    </profiles>
```

Comment:
Can you remove this empty profiles block?

Reply:
Done